data_policy.md

```diff
@@ -66,7 +68,11 @@
 
 3.4.1 Raw data and metadata beyond the period that it is stored on instrument-related computers will long-term be accessible and searchable from the on-line catalogue.
 
-3.4.2 Access to raw data and the associated metadata obtained from an experiment is restricted to the experimental team for an embargo period of three years after the end of the experiment. Thereafter, it will become available to the wider community under the CC-BY license that obliges users to acknowledge ISIS and cite [Digital Object Identifiers (DOI)](https://www.isis.stfc.ac.uk/Pages/Digital-Object-Identifiers-%28DOIs%29-for-ISIS-Data.aspx) that are linked to ISIS data. Any PI that wishes their data to remain ‘restricted access’ for a longer period will be required to make a special case to the Director of ISIS.  Any PI who wishes their data to be made publicly accessible before the end of the 3 year embargo period can ask ISIS to enable this.
+3.4.2 Access to raw data obtained from an experiment is restricted to the experimental team for an embargo period of three years after the end of the experiment. Thereafter, it will become available to the wider community under the CC-BY license that obliges users to acknowledge ISIS and cite [Digital Object Identifiers (DOI)](https://www.isis.stfc.ac.uk/Pages/Digital-Object-Identifiers-%28DOIs%29-for-ISIS-Data.aspx) that are linked to ISIS data. Any PI that wishes their data to remain ‘restricted access’ for a longer period will be required to make a special case to the Director of ISIS.
+
+3.4.3 Some elements of the metadata associated with an experiment are highlighted as being public during the proposal process, and will be publicly accessible as soon as the proposal is accepted. All other elements of the associated metadata are handled in the same way as the raw experimental data in regards to the embargo period.
```

Comment:
```suggestion
3.4.3 Some elements of the metadata associated with successful ISIS proposals are made publicly accessible as soon as the proposal is accepted such as the title of the proposal and the principle investigator. All other elements of the associated metadata are handled in the same way as the raw experimental data in regards to the embargo period.
```

Reply:
I'd like to keep it explicit that the 'source of truth' for which fields this applies to comes from the proposals system, so we don't have to update the policy if it changes. A midway point might be something like:

```suggestion
3.4.3 During the proposals process, some elements of the metadata associated with an experiment (such as the title) are indicated as being public; these will be publicly accessible immediately upon the proposal being accepted. All other elements of the associated metadata are handled in the same way as the raw experimental data in regards to the embargo period.
```